@navikt/core/css/form/combobox.css

```diff
@@ -280,6 +319,15 @@
   padding-left: calc(var(--a-spacing-2) - 4px);
 }
 
+.navds-combobox__list-item:not(
+    .navds-combobox__list-item--max-selected,
+    .navds-combobox__list-item--loading,
+    .navds-combobox__list-item--no-options
+  )[data-no-focus="true"] {
```

Comment:
```suggestion
.navds-combobox__list-item[data-no-focus="true"]:not(
    .navds-combobox__list-item--max-selected,
    .navds-combobox__list-item--loading,
    .navds-combobox__list-item--no-options
  ) {
```

Reply:
Kan denne forenkles til dette? `.navds-combobox__list-options .navds-combobox__list-item[data-no-focus="true"]`

Reply:
Fjernet `:not` lokalt